lib/pure/parsecfg.nim

```diff
@@ -370,7 +370,10 @@ proc getSymbol(c: var CfgParser, tok: var Token) =
     add(tok.literal, c.buf[pos])
     inc(pos)
     if not (c.buf[pos] in SymChars): break
-  tok.literal = tok.literal.strip
+
+  while tok.literal.len > 0 and tok.literal[^1] == ' ':
+    tok.literal.setLen(tok.literal.len - 1)
```

Comment:
doesn't handle things like `\t`
This should be instead:
```nim
tok.literal = tok.literal.strip(leading = false)
```

Reply:
`\t` is already handled by `parseSymbol`, because `\t` isn't in `SymbolChars`.

Reply:
ok. But other point remains, `tok.literal = tok.literal.strip(leading = false)` is more readable, no?

Reply:
AFAIR, `strip` inplace is used in some places in compiler, I hope it could be added to `strutils`.